src/data/bool.lean

```diff
@@ -145,3 +145,11 @@ lemma bxor_iff_ne : ∀ {x y : bool}, bxor x y = tt ↔ x ≠ y := dec_trivial
 lemma bnot_inj : ∀ {a b : bool}, !a = !b → a = b := dec_trivial
 
 end bool
+
+instance : decidable_linear_order bool :=
```

Comment:
Can this not be a global instance? This seems a bit weird, I don't know many languages other than C where you can compare booleans (and in that case it is only because C is too weakly typed and thinks they are integers). Also a better proposition for this would be `a -> b`.

Reply:
Sure, I don't mind, although I don't think it does any harm. I was surprised that there was no lattice instance already that would give this comparison.

Reply:
I agree with Rob that `bool` should have an order, in particular `bool` should be a `complete_boolean_algebra`.  Even Haskell has an order on `Bool`: https://hackage.haskell.org/package/base-4.12.0.0/docs/Data-Ord.html#i:ic:Ord:Ord:1